apps/docs/src/guide/contracts/cost-estimation.md

```diff
@@ -1,17 +1,17 @@
 # Estimating Contract Call Cost
 
-The `getTransactionCost` function provided by the [Account](../../api/Account/Account.md) allows you to estimate the cost of a specific contract call. The return type, `TransactionCost`, is an object containing relevant information for the estimation:
+The [`Account.getTransactionCost`](../../api/Account/Account.md#gettransactioncost) method allows you to estimate the cost of a specific contract call. The return type, `TransactionCost`, is an object containing relevant information for the estimation:
```

Comment:
Should we use `Contract.getTransactionCost` here instead ( `BaseInvocationScope` ) since it is the method used by contracts?

Reply:
Updated to `FunctionInvocationScope.getTransactionCost` in https://github.com/FuelLabs/fuels-ts/pull/3366/commits/0c6057b31a5e556cb3f90e727093201397dbff0a because `BaseInvocationScope` doesn't have its own api docs.